domain/infosync/info.go

```diff
@@ -78,6 +86,8 @@ type ServerInfo struct {
 	Lease          string `json:"lease"`
 	BinlogStatus   string `json:"binlog_status"`
 	StartTimestamp int64  `json:"start_timestamp"`
+
+	Host string `json:"-"`
```

Comment:
Do we need this field? (can we use the `IP` field instead of adding a new field)